luatest/server.lua

```diff
@@ -130,12 +131,32 @@ function Server:start()
 end
 
 --- Restart server process.
-function Server:restart(args)
+function Server:restart(params)
+    checks('table', {
+        command = '?string',
+        workdir = '?string',
+        chdir = '?string',
+        env = '?table',
+        args = '?table',
+
+        http_port = '?number',
+        net_box_port = '?number',
+        net_box_uri = '?string',
+        net_box_credentials = '?table',
+
+        alias = '?string',
+
+        coverage_report = '?string',
+    })
     if not self.process then
-        log.warn("Process isn't running")
+        log.warn("Process hasn't started")
```

Comment:
I think the previous message was more accurate. Or even "Process wasn't started".